scripts/setup/upgrade-postgres

```diff
@@ -43,12 +43,22 @@ pg_ctlcluster "$UPGRADE_FROM" main stop
     rm -rf "$TEMP_CONF_DIR"
 )
 
-pg_upgradecluster "$UPGRADE_FROM" main
+# Capture the output so we know where the path to the post-upgrade scripts is
+UPGRADE_LOG=$(mktemp)
```

Comment:
Do we want to just write to `/var/log/zulip/postgres-upgrade.XXXXXXXXX.log`  as template if we're not deleting this?